src-self-hosted/llvm.zig

```diff
@@ -288,6 +288,6 @@ extern fn ZigLLVMTargetMachineEmitToFile(
 ) bool;
 
 pub const BuildCall = ZigLLVMBuildCall;
-extern fn ZigLLVMBuildCall(B: *Builder, Fn: *Value, Args: [*]*Value, NumArgs: c_uint, CC: c_uint, fn_inline: CallAttr, Name: [*:0]const u8) ?*Value;
+extern fn ZigLLVMBuildCall(B: *Builder, Fn: *Value, Args: [*]*Value, NumArgs: c_uint, CC: CallConv, fn_inline: CallAttr, Name: [*:0]const u8) ?*Value;
```

Comment:
Changing `CallConv` back to `c_uint` causes an assertion failure instead of a compile error.

Reply:
Let me try it real quick and see if it's an easy fix.

Reply:
ad92227516bf977a3be2802db525ac2d678acc7c fixes this particular issue. I wasn't able to come up with a small reproducing test case, but at least the fix is in master now.